src/pages/api/paymentTxParams/[uuid].ts

```diff
@@ -11,47 +13,25 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     try {
       const { uuid, buyerAddress } = req.query;
 
-      if (typeof uuid !== 'string') {
+      if (!uuid || typeof uuid !== 'string') {
         return res.status(400).json({ message: 'Invalid UUID' });
       }
 
+      // TODO: Justin / Rafa is this stil buyer? or sender? I dont remember
```

Comment:
this should be `senderAddress`, I haven't updated it yet

Reply:
OK I can change here